apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -18,6 +18,7 @@ import TouchSensor from './TouchSensor';
 import Piezo from './Piezo';
 import NeoPixel from './NeoPixel';
 import Led from './Led';
+import experiments from '../../../util/experiments';
```

Comment:
do we have a standard for when we use `@cdo` instead of relative pathing? Because this seems like a good place to do that